__tests__/app.test.js

```diff
@@ -138,6 +138,48 @@ describe("API Articles", () => {
         });
     });
   });
+  describe("GET /api/articles?:topic", () => {
+    it('200 should return articles filtered by topic', () => {
+      return request(app)
+      .get('/api/articles?topic=mitch')
+      .expect(200)
+      .then(({body}) => {
+        const articles = body.articles
+        const firstArticle = articles[0]
+        expect(articles.length).toBe(12)
```

Comment:
A good test would also be to iterate through every article that comes back from the api and make sure that their topic is mitch